incremental/playbooks/create-cell0.yml

```diff
@@ -0,0 +1,44 @@
+---
+
+- name: Ensure nova_cell0 database is created
+  hosts: "galera[0]"
+  user: root
+  gather_facts: false
+  tasks:
+    - name: Ensure nova_cell0 is created
+      mysql_db:
+        login_user: "{{ galera_root_user }}"
+        login_password: "{{ galera_root_password }}"
+        login_host: "{{ nova_galera_address }}"
+        name: "{{ nova_cell0_database }}"
+        state: present
+
+    - name: Grant access to DBs for the services
+      mysql_user:
+        login_user: "{{ galera_root_user }}"
+        login_password: "{{ galera_root_password }}"
+        login_host: "{{ nova_galera_address }}"
+        name: "{{ item['name'] }}"
+        password: "{{ item['password'] }}"
+        host: "{{ item['host'] }}"
+        state: "present"
+        priv: "{{ item['database'] }}.*:ALL"
+        append_privs: "{{ item['db_append_privs'] | default(omit) }}"
+      with_items:
```

Comment:
Could the with_items clause be simplified to only list the host entries?

Reply:
Yeah, it probably could.  I copied some of this from upstream and it was set to do additional items as well so there's probably a few things we could compress.  It worked though so I haven't tried optimizing it yet.